### PR TITLE
Fix performances links spaces and URLs

### LINKED ISSUE #28227: Links redirect to 404 pages and spaces missing in wording
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Spaces are missing on "APC" and "Xcache" label in caching block:

This is a regression compared to PS1778

Furthermore, these two links, "[APC PECL extension](http://www.php.net/manual/en/apc.installation.php)" and "[Xcache extension](http://xcache.lighttpd.net/)" redirect to 404 pages:

https://user-images.githubusercontent.com/2854834/162772349-4d8463ab-0912-4866-b64f-640d50d86de3.mp4

### Expected behavior

On 1778 version, spaces were good: 

But the links were not good too.
I've found the good one for APC link. It should be: https://www.php.net/manual/en/apcu.installation.php
For Xcache, couldn't find it.

### Steps to reproduce

1. Go to BO -> Advanced parameters -> performance -> caching block
2. Set "Use cache" to yes and the caching system list appears.

### PrestaShop version(s) where the bug happened

1.7.8.0

### PHP version(s) where the bug happened

7.2

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | I added a small margin to links of the Performance page in order to have a space and fixed links
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #28227.
| How to test?      | See issue
| Possible impacts? | Performance page links and spaces

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
Nice catch for 404

File: src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php [hunk 1]
```diff
@@ -128,7 +128,7 @@ private function getErrorsMessages()
                     '(you must install the [a]Memcached PECL extension[/a])',
                     'Admin.Advparameters.Notification',
                     [
-                        '[a]' => '<a href="http://www.php.net/manual/en/memcached.installation.php" target="_blank">',
+                        '[a]' => '<a href="http://www.php.net/manual/en/memcached.installation.php" class="ml-1" target="_blank">',
```
Comment (Progi1984) [inline src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php]:
```suggestion
                        '[a]' => '<a href="https://www.php.net/manual/en/memcached.installation.php" class="ml-1" target="_blank">',
```

File: src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php [hunk 2]
```diff
@@ -118,7 +118,7 @@ private function getErrorsMessages()
                     '(you must install the [a]Memcache PECL extension[/a])',
                     'Admin.Advparameters.Notification',
                     [
-                        '[a]' => '<a href="http://www.php.net/manual/en/memcache.installation.php" target="_blank">',
+                        '[a]' => '<a href="http://www.php.net/manual/en/memcache.installation.php" class="ml-1" target="_blank">',
```
Comment (Progi1984) [inline src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php]:
```suggestion
                        '[a]' => '<a href="https://www.php.net/manual/en/memcache.installation.php" class="ml-1" target="_blank">',
```

**Review by Progi1984 — COMMENTED** [review]
Two small changes

**Review by Progi1984 — DISMISSED** [review]

**Review by ghost — DISMISSED** [review]

**Review by sLorenzini — DISMISSED** [review]
hello @NeOMakinG,

Thank you for this PR.
Tested on 1.7.8x branch, PHP 7.3

Links are good and wording have their margins.

QA approved ✅ 

Thanks

**Review by kpodemski — APPROVED** [review]

**Review by kpodemski — DISMISSED** [review]
Hi @NeOMakinG 

It's a regression, I think you wanted to target `1.7.8` branch?

**Comment by NeOMakinG** [comment]
> Hi @NeOMakinG
> 
> It's a regression, I think you wanted to target `1.7.8` branch?

Fuck my life :p

**Review by ghost — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @NeOMakinG

**Pull request merged**